SECURITY.md

```diff
@@ -0,0 +1,38 @@
+# Security
+
+As part of our [Coordinated Vulnerability Disclosure
+Policy](https://tendermint.com/security), we operate a bug bounty.
+See the policy for more details on submissions and rewards.
+
+The following is a list of examples of the kinds of bugs we're most interested in for
+the cosmos-sdk. See [here](https://github.com/tendermint/tendermint/blob/master/SECURITY.md) for vulnerabilities we are interested in for tendermint / lower level libs.
+
+## Specification
+- Conceptual flaws
```

Comment:
Vague - e.g. I think the reserve pool may be a conceptual flaw, https://github.com/cosmos/cosmos-economics/issues/32, but that's definitely not the kind of issue we want submitted to the bug bounty

Mostly we should bounty differences between the spec and the implementation IMO

Reply:
Not sure I agree here. I agree that the reserve pool is a conceptual flaw, but I think getting that reported as a crypto-economic security issue would be a good thing. 

I take you're point though, that could equivalently be done through another program, perhaps a research program. I feel like conceptual flaws in the specification can be called bugs / are good to have under security reports though.

Reply:
We also use conceptual flaws in tendermint's `SECURITY.md` as well.